crates/containerd-shim-wasm/src/sandbox/cli.rs

```diff
@@ -48,7 +86,18 @@ pub fn shim_main<'a, I>(
     I: 'static + Instance + Sync + Send,
     I::Engine: Default,
 {
+    #[cfg(feature = "opentelemetry")]
```

Comment:
why not put this code in `fn shim_main_with_otel` where this is already guarded by this feature?

Reply:
Oh it's because `shim_main_with_otel` does not have a span yet, and thus `OtelConfig::get_trace_context()` will return an empty string for the context. 

We could add span to `shim_main_with_otel` though but that will add one more layer of tracing, which I don't think it's bad necessarily.